cmd/operator/internal/controller/kubearchiveconfig_controller.go

```diff
@@ -96,37 +107,43 @@ func (r *KubeArchiveConfigReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{}, nil
 	}
 
-	_, err := r.reconcileA14eServiceAccount(ctx, kaconfig)
+	cm, err := r.reconcileFilterConfigMap(ctx, kaconfig)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+	resources := r.parseConfigMap(ctx, cm)
+
+	_, err = r.reconcileA14eServiceAccount(ctx, kaconfig)
```

Comment:
`kaconfig` is not used.

```suggestion
	_, err = r.reconcileA14eServiceAccount(ctx)
```

Reply:
Fixed.